opentok/opentok.py

```diff
@@ -1127,3 +1130,26 @@ def _create_jwt_auth_header(self):
         }
 
         return jwt.encode(payload, self.api_secret, algorithm="HS256")
+
+
+class OpenTok(Client):
+    def __init__(
+        self,
+        api_key,
+        api_secret,
+        api_url="https://api.opentok.com",
+        timeout=None,
+        app_version=None,
+    ):
+        warnings.warn(
+            "OpenTok class is deprecated (Use Client class instead)",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        super(OpenTok, self).__init__(
+            api_key,
+            api_secret,
+            api_url="https://api.opentok.com",
```

Comment:
named parameters here shouldn't be hardcoded in the superclass's constructor call. They are being passed into this constructor (and given a default value) so this would essentially be overriding any updated values the user passes in.

Reply:
@slorello89  what do you mean? the `api_url` with values passed in here would overwrite what users write? a quick reference:

```
Python 3.7.6 (default, Dec 31 2019, 13:13:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def method(parameter='hola mundo'):
...     print(parameter)
... 
>>> method()
hola mundo
>>> method('adios mundo cruel')
adios mundo cruel
>>> 
```

Reply:
yeah, so in this case we're calling the parent class's constructor within our constructor - which is fine - however the api_url, which can be passed into our constructor - will be overridden by the hard coded http://api.opentok.com - because we're explicitly setting it as a named argument - my only suggestion here is to remove the hard coded string - which I think will override what the user passes in.